site/en/tutorials/images/data_augmentation.ipynb

```diff
@@ -0,0 +1,599 @@
+        "  image = tf.image.random_crop(image, size=[28,28,1]) #providing random crop to image\n",
+        "  image = tf.image.random_brightness(image, max_delta=0.5) #providing random brightness to image\n",
+        "  image = tf.image.random_flip_left_right(image) #providing random flip to image\n",
+        "\n",
+        "  return image,label\n",
+        "\n",
+        "BATCH_SIZE = 64\n",
+        "\n",
+        "train_batches = train_dataset.shuffle(num_train_examples//4).map(scale).batch(BATCH_SIZE).prefetch(1) #A batch dataset that can be directly passed to model for training"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Yi9TIwR-ZIOi",
+        "colab_type": "text"
+      },
+      "source": [
+        "Creating and compiling the model. The model will be a 2 layered Fully connected neural network without convolution."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "hHhkA4Q0CsHx",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "model = tf.keras.Sequential([\n",
+        "            layers.Flatten(input_shape=(28, 28,1)),\n",
+        "            layers.Dense(256, activation='relu'),\n",
+        "            layers.Dense(128, activation='relu'),\n",
+        "            layers.Dense(10, activation='softmax')\n",
+        "])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "17wqPAAoNe3N",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "model.compile(optimizer = 'adam',\n",
+        "                    loss='sparse_categorical_crossentropy',\n",
+        "                    metrics=['accuracy'])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "P0rciou3ZWwy",
+        "colab_type": "text"
+      },
+      "source": [
+        "Training the model"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "z8X8CpqvNhG9",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "model.fit(train_batches, epochs=5)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UEqeeNsHZaC5",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Conclusion:\n",
+        "As we can see, the model provided 96.31% accuracy on training set. This is slightly higher than the model trained without data augmentation. So, Data Augmentation is good. It didn't provide much significance on this model because the dataset already had a large number of samples. But, on a small dataset, you could see a huge difference."
```

Comment:
```suggestion
        "This model provides ~95% accuracy on training set. This is slightly higher than the model trained without data augmentation. It didn't provide much significance on this model because the dataset already had a large number of samples. But, on a small dataset, you could see a huge difference."
```

I'm seeing 95.8 in my run so we don't seen so be so specific in the text